src/index.js

```diff
@@ -0,0 +1,83 @@
+import './style.css';
+
+const tasks = [
```

Comment:
Please make sure that the tasks appear on the screen in order of the index values for each task. The user is unable to see a list of tasks on the screen at the moment.